modules/ocf_desktop/files/xsession/lab_close_notify

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import datetime
+import sched
+import subprocess
+import time
+
+from ocflib.lab import hours
+
+# 0 minutes, 2 minute, 5 minutes, 10 minutes, 15 minutes
+WARNING_DURATIONS = [0, 120, 300, 600, 900]
+
+def notify_user(seconds_left):
+    title = 'Lab closure'
+    if seconds_left != 0:
+        msg = 'Lab closing in {0:g} minutes'.format(seconds_left / 60)
+    else:
+        msg = 'Lab is closing NOW!'
+
+    # TODO: pick an icon
+    subprocess.call(['notify-send', title, msg])
+
+today = datetime.datetime.today()
+todays_hours = hours.REGULAR_HOURS[(today.weekday() - 1) % 7]
```

Comment:
This won't take into account holidays when the lab is sometimes open but with adjusted hours. I think you can use this instead:

```python
from ocflib.lab.hours import Day

...

todays_hours = Day.from_date().hours
```

That should do the right thing on both regular days and holidays.